docs/modules/general/pages/bootstrapping.adoc

```diff
@@ -0,0 +1,567 @@
+= OneCX Microfrontend Content Bootstrapping
+
+== Overview
+// TODO: Add link to MFE loading
+This document aims to explain the details related to Microfrontend's content bootstrapping. Depending on the content's type and expose method there are different requirements needed to be fullfield to ensure the content is loaded correctly by OneCX platform elements. This topic is highly related to Microfrontend content loading and xref:webpack.adoc[Microfrontend configuration with Webpack]. It is recommended to be familiar with those before reading this document.
+
+== Microfrontend's Content Bootstrapping Explained
```

Comment:
OneCX Microfrontend ~~'s~~ Content Bootstrapping --> Microfrontend's Content or Microfrontend Content? Probably means the same at the end, the latter may be easier to read